active_model_otp.gemspec

```
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.3"
 
   spec.add_dependency "activemodel"
-  spec.add_dependency "rotp", "~> 5.0.0"
+  spec.add_dependency "rotp", "~> 5"
```

```suggestion
  spec.add_dependency "rotp", "~> 5.1"
```
Equivalent to > 5.1 and < 6.0.  By convention 6.0 would have breaking changes, but 5.1 includes bug fixes (that I want).